docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,104 @@
 
-> To make the process less tedious you can apply a filter in settings to show only the server.py process in the list. To do that navigate to: `Settings/Preferences | Build, Execution, Deployment | Python Debugger` and then in `Attach to process` field put in: `twistd.exe" --nodaemon`. This is an example for windows, I don't have a working mac/linux box.
 
-![Example process filter configuration](https://i.imgur.com/vkSheR8.png)
-
-## Run Evennia from inside PyCharm
+### Run Evennia with a Run/Debug Configuration
 
 This configuration allows you to launch Evennia from inside PyCharm. Besides convenience, it also allows suspending and debugging the evennia_launcher or evennia_runner at points earlier than you could by running them externally and attaching. In fact by the time the server and/or portal are running the launcher will have exited already.
 
+#### On Windows
 1. Go to `Run > Edit Configutations...`
-1. Click the plus-symbol to add a new configuration and choose Python
-1. Add the script: `\<yourrepo\>\evenv\Scripts\evennia_launcher.py` (substitute your virtualenv if it's not named `evenv`)
-1. Set script parameters to: `start -l` (-l enables console logging)
-1. Ensure the chosen interpreter is from your virtualenv
-1. Set Working directory to your `mygame` folder (not evenv nor evennia)
-1. You can refer to the PyCharm documentation for general info, but you'll want to set at least a config name (like "MyMUD start" or similar).
+2. Click the plus-symbol to add a new configuration and choose Python
+3. Add the script: `\<yourprojectfolder>\.venv\Scripts\evennia_launcher.py` (substitute your virtualenv if it's not named `evenv`)
+4. Set script parameters to: `start -l` (-l enables console logging)
+5. Ensure the chosen interpreter is your virtualenv
+6. Set Working directory to your `mygame` folder (not your project folder nor evennia)
+7. You can refer to the PyCharm documentation for general info, but you'll want to set at least a config name (like "MyMUD start" or similar).
 
-Now set up a "stop" configuration by following the same steps as above, but set your Script parameters to: stop (and name the configuration appropriately).
+A dropdown box holding your new configurations should appear next to your PyCharm run button. 
+Select it start and press the debug icon to begin debugging.
 
-A dropdown box holding your new configurations should appear next to your PyCharm run button. Select MyMUD start and press the debug icon to begin debugging.  Depending on how far you let the program run, you may need to run your "MyMUD stop" config to actually stop the server, before you'll be able start it again.
+#### On Linux
+1. Go to `Run > Edit Configutations...`
+2. Click the plus-symbol to add a new configuration and choose Python
+3. Add the script: `/<yourprojectfolder>/.venv/bin/twistd` (substitute your virtualenv if it's not named `evenv`)
```

Comment:
This still lists `.venv` in the path which is confusing.

Reply:
Sorry, will fix that